test/yaml/iperf3-vclsidecar/test.yaml

```diff
@@ -0,0 +1,125 @@
+apiVersion: apps/v1
+kind: ReplicaSet
+metadata:
+  namespace: iperf3-vclsidecar
+  name: iperf3-server-repl
+  labels:
+    app: iperf3-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: iperf3-server
+  template:
+    metadata:
+      labels:
+        app: iperf3-server
+      annotations:
+        "cni.projectcalico.org/vppVcl": "enable"
+    spec:
+      volumes:
+        - name: config-vcl
+          configMap:
+            name: vcl
+        - name: libraries
+          emptyDir: {}
+      containers:
+      - name: iperf3-server
+        image: calicovpp/test-iperf3:v3.29.0
+        command: ["tail", "-f", "/dev/null"]
+        env:
+        - name: VCL_CONFIG
+          value: /vcl/vcl.conf
+        - name: LD_LIBRARY_PATH
+          value: /libraries           
+        volumeMounts:
+          - name: config-vcl
+            mountPath: /vcl
+          - name: libraries
+            mountPath: /libraries              
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 5201
+      - name: vclsidecar
+        image: docker.io/calicovpp/vclsidecar:v3.29.0
+        command: ["/bin/sh", "-c"]
+        args:
+          - cp /usr/local/lib/* /libraries && sleep infinity
```

Comment:
We can maybe simplify this using something like this [0]
which would prevent us from having to copy
```yaml
spec:
  containers:
  - name: shell
    command: ["sleep", "infinity"]
    image: debian
    volumeMounts:
    - name: volume
      mountPath: /volume
  volumes:
  - name: volume
    image:
      reference: quay.io/crio/artifact:v1
      pullPolicy: IfNotPresent
```
[0] https://kubernetes.io/docs/concepts/storage/volumes/#image

Reply:
This is actually supported in Kubernetes v1.33 and still in Beta, I don't think we can use it for now, especially for current customer :)